tests/installation/user_settings.pm

```diff
@@ -35,12 +35,19 @@ sub run {
     }
 
     $self->enter_userinfo();
-    assert_screen 'inst-userinfostyped';
-    if (match_has_tag 'boo#1122804') {
-        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
-        $self->enter_userinfo(retry => 1);
-        assert_screen 'inst-userinfostyped';
+    assert_screen 'inst-userinfostyped-ignore-full-name';
+    for (my $i = 0; $i < 3; $i++) {
```

Comment:
I don't see how the for-loop could help here as we have an `assert_screen` in line 47 so if there is mistyping even in the retry I doubt you would go into another iteration.